static.config.js

```diff
@@ -85,8 +88,37 @@ const makeServicePages = async client => {
   return data;
 };
 
+const makeProcessPages = async client => {
+  const { allProcesses } = await client.request(allProcessesQuery);
+  const processes = cleanProcesses(allProcesses);
+
+  const data = {
+    path: '/processes',
+    component: 'src/components/Pages/Processes',
+    getData: async () => ({
+      processes,
+    }),
+    children: processes.map(process => ({
+      path: `/${process.slug}`,
+      component: 'src/components/Pages/Process',
+      getData: async () => ({
+        process,
+      }),
+      children: process.processSteps.map(processStep => ({
+        path: `/${processStep.slug}`,
```

Comment:
Nice find, I didn't know that `Orderable`'s have slugs. This seems better than using `sortOrder`.

Reply:
Oh it doesn't -- but I really wanted to keep the same format for all the page data, so I'm building the slug in cleanProcesses(allProcesses)